src/conversions.jl

```diff
@@ -616,6 +616,29 @@ function utctai(jd1, jd2)
     date, date1
 end
 
+"""
+   utcut1(jd1, jd2)
+
+Transform a two-part Julian date from `UTC` to `UT1`.
+
+# Example
+
+```jldoctest
+julia> utc  = Epoch{UTC}(2.4578265e6, 0.30477440993249416)
+2017-03-14T07:18:52.509 UTC
+julia> AstroTime.Epochs.utcut1(utc.jd1, utc.jd2)
+(2.4578265e6, 0.30477440993249416)
+```
+"""
+@inline function utcut1(jd1, jd2, dut1)
+    jd = +(jd1, jd2)
+    dat = leapseconds(jd)
```

Comment:
`dat` should become a parameter so that users could use another leap seconds source if desired, e.g. `utcut1(jd1, jd2, dut1, dat)`